### PR TITLE
[FIX] - Fix flash message for incorrect login

### PR DESCRIPTION
**Pull request by mkoura**
In 5.6 the new message is "Incorrect username or password".

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 124703b7a1d611ddffa259d33362eb6d05385e54:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/test_login.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ def test_bad_password():
     creds = Credential(principal=conf.credentials['default']['username'], secret="badpassword@#$")
     user = User(credential=creds)
 
-    with error.expected('Sorry, the username or password you entered is incorrect.'):
+    with error.expected(version.pick({version.LOWEST:
```
Comment (psav) [inline cfme/tests/test_login.py]: Stylistically we'd probably prefer this
```python
    with error.expected(version.pick({
        version.LOWEST: "Sorry, the username or password you entered is incorrect.",
        "5.6": "Incorrect username or password"})):

**Comment by psav** [comment]
Waiting on PRT run

**Comment by dajoRH** [comment]
Lint report for commit 2ea1eb003d824593cf05e2fc0a786c88c7e50af1:

`cfme/tests/test_login.py`:
- :bangbang: Line 46:9: **E129** *visually indented line with same indent as next logical line*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dae2bbac0b8d72913f77f1a5c0067049353fbb66:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3cb07277bbd8e4cb22d1e4b403e274d058288502:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
@mkoura First PR, right?

**Pull request merged**